client/stylesheets/_toCat.scss

```diff
@@ -4064,6 +4068,10 @@ On click the images will be displayed at normal size to complete the effect
     border: 2px solid #f8504b;
 }
 
+.profileCardInfo {
+    text-align: left !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces